Source/WebCore/PAL/pal/crypto/commoncrypto/CryptoDigestCommonCrypto.cpp

```diff
@@ -190,19 +190,19 @@ String CryptoDigest::toHexString()
     return String::fromUTF8(result);
 }
 
-std::optional<Vector<uint8_t>> CryptoDigest::computeHash(CryptoDigest::Algorithm algo, const Vector<uint8_t>& data, bool useCryptoKit)
+std::optional<Vector<uint8_t>> CryptoDigest::computeHash(CryptoDigest::Algorithm algo, Vector<uint8_t>&& data, bool useCryptoKit)
```

Comment:
Why switch to a r-value reference? Computing a hash shouldn't require adopting the vector.